app/enterprise/1.3-x/deployment/installation/amazon-linux-2.md

```diff
@@ -1,343 +1,320 @@
---- 
-title: Install Kong Enterprise with Amazon Linux 2
+---
+title: Install Kong Enterprise on Amazon Linux 2
 ---
 
 ## Introduction
 
-This guide will walk you through installing and running **Kong Enterprise** on an existing **Amazon Linux 2 EC2** instance.
-
-**Supported Database Options:**
-
-* PostgreSQL 9.6, 10, and 11
-* Cassandra 3.11
+This guide walks through downloading, installing, and starting **Kong Enterprise** on **Amazon Linux 2**.
 
-Due to the nature of setting up a Kong cluster with Cassandra, this documentation only covers Postgres. For assistance in setting up Cassandra, please contact your Sales or Support representative.
+The configuration shown in this guide is intended only as an example. Depending on your
```

Comment:
remove "only" ....`The configuration shown in this guide is intended as an example`